cmds/ocm/commands/ocmcmds/common/cmds/signing/cmd.go

```diff
@@ -89,7 +91,9 @@ func (o *SignatureCommand) Run() (rerr error) {
 	handler := comphdlr.NewTypeHandler(o.Context.OCM(), session, repo, comphdlr.OptionsFor(o))
 	sopts := signing.NewOptions(sign, signing.Resolver(repo, lookup.Resolver))
 	if !o.spec.sign {
-		sopts.VerifySignature = true
+		if len(sopts.SignatureNames) > 0 || sopts.Issuer != nil || sopts.Keyless {
```

Comment:
What's the meaning of `Keyless`? It looks odd: "if there is no key (aka. Keyless), then verify the signature" ... I don't get it